javascript/uber_search.js

```diff
@@ -248,16 +277,19 @@ var UberSearch = function(data, options){
   }
 
   function buildResult(datum){
-    var result = $('<li class="result"></li>')
-      .html((options.treatBlankOptionAsPlaceholder ? datum.text || options.placeholder : datum.text) || "&nbsp;")
-      .data(datum) // Store the datum so we can get know what the value of the selected item is
+    // var result = $('<li class="result"></li>')
+    //   .html((options.treatBlankOptionAsPlaceholder ? datum.text || options.placeholder : datum.text) || "&nbsp;")
+    //   .data(datum) // Store the datum so we can get know what the value of the selected item is
+
+    var $link   = $('<a href="#"></a>').html((options.treatBlankOptionAsPlaceholder ? datum.text || options.placeholder : datum.text) || "&nbsp;")
```

Comment:
Why do we need to place this in a link? Can't the content go directly into the list item? Since the `<li>` has a tab index, it should be able to receive focus events.

Reply:
@njakobsen See my reply to yours on line 97.